[nightly] runtime Error tasmota

### Describe the bug

aktuelle nightly hat wohl ein Problem mit tasmota charger.

### Steps to reproduce

yaml mit konfigurierten tasmota-charger

### Configuration details

```yaml
network:
  schema: http
  host: evcc
  port: 7070
interval: 10s

influx:
  url: http://192.168.1.2:8086
  database: evcc
  user: *****
  password: *****

mqtt:
  broker: 192.168.1.2:1883
  topic: evcc
  user: *****
  password: *****

log: debug
levels:
  #site: trace
  #salia: trace
  lp-2: info

vehicles:
  - name: my_peugeot
    type: template
    template: flobz
    title: e-208
    url: http://192.168.1.2:5050
    vin: *****
    capacity: 46
    #mode: pv
    #targetSoC: 80
  - name: my_kia
    type: custom
    title: Niro-EV
    soc:
      source: mqtt
      topic: nodered/evnotify/all
      jq: .soc_display
    status:
      source: mqtt
      topic: nodered/evnotify/status
    range:
      source: mqtt
      topic: nodered/evnotify/all
      jq: .range
    #akey: c53705
    #token: *****
    capacity: 64.8
  - name: my_dummy
    type: template
    template: offline
    title: evDummy

chargers:
  - name: my_charger
    type: template
    template: hardybarth-salia
    host: 192.168.1.11
  - name: psa_charger
    type: template
    template: tasmota
    host: 192.168.2.84 # IP-Adresse oder Hostname
    user: *****
    password: *****
    channel: 1 # Nummer des Schaltkanals (1-8), bei Geräten mit mehr als einem Schalter
    standbypower: 15 # Leistung oberhalb des angegebenen Wertes wird als Ladeleistung gewertet (optional)

meters:
  - name: my_grid
    type: template
    template: victron-energy
    usage: grid
    host: 192.168.1.3
  - name: my_pv
    type: template
    template: victron-energy
    usage: pv
    host: 192.168.1.3
  - name: my_battery
    type: template
    template: victron-energy
    usage: battery
    host: 192.168.1.3
    capacity: 14.34

site:
  title: Home
  meters:
    grid: my_grid
    pv:  my_pv
    battery: my_battery
  residualPower: 100

loadpoints:
  - title: Carport
    charger: my_charger
    mode: pv
    phases: 3
    mincurrent: 6
    maxcurrent: 16
    soc:
      poll:
        mode: connected
        interval: 10m
  - title: PSA-Charger
    charger: psa_charger
    vehicle: my_dummy
    mode: off
    phases: 1
    maxcurrent: 10
    mincurrent: 10

tariffs:
  grid:
    #type: fixed
    #price: 0.3355
    type: tibber
    token: *****
  feedin:
    type: fixed
    price: 0.0736

plant: *****
sponsortoken: *****
```


### Log details

```text
[main  ] INFO 2023/12/31 08:53:53 evcc 0.123.2 (cf597421)
[main  ] INFO 2023/12/31 08:53:53 using config file: /etc/evcc.yaml
[main  ] INFO 2023/12/31 08:53:53 starting ui and api at :7070
[db    ] INFO 2023/12/31 08:53:53 using sqlite database: /root/.evcc/evcc.db
[mqtt  ] INFO 2023/12/31 08:53:53 connecting evcc-1217674723 at tcp://192.168.1.2:1883
[mqtt  ] DEBUG 2023/12/31 08:53:53 tcp://192.168.1.2:1883 connected
[salia ] TRACE 2023/12/31 08:53:53 GET http://192.168.1.11/api
[salia ] TRACE 2023/12/31 08:53:53 {"device":{"product":"2310007","modelname":"Salia PLCC Slave","hardware_version":"1.0","software_version":"2.0.2","vcs_version":"V0R5c","hostname":"salia","mac_address":"00:01:87:0c:7c:41","serial":"100818241","uuid":"5491ad62-022a-4356-a32c-0001870c7c41","internal_id":"391222","ip_lo":"127.0.0.1","ip_br0:fallback":"169.254.12.53","ip_br0":"192.168.1.11"},"secc":{"port0":{"salia":{"chargemode":"manual","keyswitch":"0","mem":"335296","thermal":"48165","uptime":"1 day","thermalmax":"55509","load":"0.20","thermalmin":"0","modbus":{"status":"running"},"socketminamp":"6","socketmaxamp":"16","LM":{"internal":{"status":"internal lm not activated"}},"intctrl_limit":"16","chargedata":"-","ecoplus":"0","mains_type":"none","authmode":"free","phase_switching":{"actual":"3","status":"idle"},"tc_enabled":"0","firmwareprogress":"0","firmwarestate":"idle","status":"manual -\/-\/r -\/-\/- 0(6-16) ","limit_tc":"32","extctrl_limit":"0","pausecharging":"1","heartbeat":"alive"},"ci":{"evse":{"basic":{"grid_current_limit":{"actual":"16"},"phase_count":"3","physical_current_limit":"16","offered_current_limit":"0.0","current":{"offered":"0.0","l1":{"offered":"0.0"},"l2":{"offered":"0.0"},"l3":{"offered":"0.0"}},"power":{"offered":"0.0"}},"phase":{"actual":"0"}},"ev":{"soc":{"actual":"-1","timestamp":"2023-12-29T21:45:41.788Z"}},"charge":{"cp":{"status":"A"},"plug":{"status":"unlocked"},"contactor":{"status":"opened"},"pwm":{"status":"100.00"}}},"session":{"authorization_status":"0","authorization_method":"0"},"contactor":{"state":{"hlc_target":"0","actual":"0","target":"0"},"error":"0"},"metering":{"meter":{"serialnumber":"21030651","type":"klefr","name":"Klefr 693x\/694x","phase_count":"3","supported_measurands":"Frequency,Voltage.L1-N,Voltage.L2-N,Voltage.L3-N,Current.Import.L1,Current.Import.L2,Current.Import.L3,Power.Factor,Power.Active.Import,Power.Active.Import.L1,Power.Active.Import.L2,Power.Active.Import.L3,Energy.Active.Import.Register,Energy.Active.Export.Register","available":"1"},"eichrecht_protocol":"none","mains_frequency":{"actual":"50000"},"voltage":{"ac":{"l1":{"actual":"23550"},"l2":{"actual":"23940"},"l3":{"actual":"23920"}}},"current":{"ac":{"l1":{"actual":"20"},"l2":{"actual":"0"},"l3":{"actual":"0"}}},"power_factor":{"ac":{"actual":"990"}},"power":{"active_total":{"actual":"50"},"active":{"ac":{"l1":{"actual":"50"},"l2":{"actual":"0"},"l3":{"actual":"0"}}}},"energy":{"active_total":{"actual":"6393240"},"active_import":{"actual":"6393240"},"active_export":{"actual":"0"}},"timestamp":"2023-12-31T07:53:47.104Z"},"emergency_shutdown":"0","rcd":{"feedback":{"available":"1"},"state":{"actual":"1"},"recloser":{"available":"0"}},"rfid":{"available":"0","authorizereq":""},"availability":{"actual":"operative"},"plug_lock":{"state":{"actual":"0","target":"0"},"error":"0"},"cp":{"pwm_state":{"actual":"1"},"state":"A","duty_cycle":"100.00"},"diode_present":"0","cable_current_limit":"-1","ev_present":"0","charging":"0","ready_for_slac":"0","ventilation":{"state":{"actual":"0"},"available":"0"},"charging_stop":"1"}}}
[salia ] TRACE 2023/12/31 08:53:53 PUT http://192.168.1.11/api/secc
[salia ] TRACE 2023/12/31 08:53:53 {"salia/pausecharging":"1"}
--
result=ok 
[lp-1  ] WARN 2023/12/31 08:53:53 poll mode '{connected 10m0s}' may deplete your battery or lead to API misuse. USE AT YOUR OWN RISK.
[salia ] TRACE 2023/12/31 08:53:53 GET http://192.168.1.11/api
[salia ] TRACE 2023/12/31 08:53:53 {"device":{"product":"2310007","modelname":"Salia PLCC Slave","hardware_version":"1.0","software_version":"2.0.2","vcs_version":"V0R5c","hostname":"salia","mac_address":"00:01:87:0c:7c:41","serial":"100818241","uuid":"5491ad62-022a-4356-a32c-0001870c7c41","internal_id":"391222","ip_lo":"127.0.0.1","ip_br0:fallback":"169.254.12.53","ip_br0":"192.168.1.11"},"secc":{"port0":{"salia":{"chargemode":"manual","keyswitch":"0","mem":"335296","thermal":"48165","uptime":"1 day","thermalmax":"55509","load":"0.20","thermalmin":"0","modbus":{"status":"running"},"socketminamp":"6","socketmaxamp":"16","LM":{"internal":{"status":"internal lm not activated"}},"intctrl_limit":"16","chargedata":"-","ecoplus":"0","mains_type":"none","authmode":"free","phase_switching":{"actual":"3","status":"idle"},"tc_enabled":"0","firmwareprogress":"0","firmwarestate":"idle","status":"manual -\/-\/r -\/-\/- 0(6-16) ","limit_tc":"32","extctrl_limit":"0","pausecharging":"1","heartbeat":"alive"},"ci":{"evse":{"basic":{"grid_current_limit":{"actual":"16"},"phase_count":"3","physical_current_limit":"16","offered_current_limit":"0.0","current":{"offered":"0.0","l1":{"offered":"0.0"},"l2":{"offered":"0.0"},"l3":{"offered":"0.0"}},"power":{"offered":"0.0"}},"phase":{"actual":"0"}},"ev":{"soc":{"actual":"-1","timestamp":"2023-12-29T21:45:41.788Z"}},"charge":{"cp":{"status":"A"},"plug":{"status":"unlocked"},"contactor":{"status":"opened"},"pwm":{"status":"100.00"}}},"session":{"authorization_status":"0","authorization_method":"0"},"contactor":{"state":{"hlc_target":"0","actual":"0","target":"0"},"error":"0"},"metering":{"meter":{"serialnumber":"21030651","type":"klefr","name":"Klefr 693x\/694x","phase_count":"3","supported_measurands":"Frequency,Voltage.L1-N,Voltage.L2-N,Voltage.L3-N,Current.Import.L1,Current.Import.L2,Current.Import.L3,Power.Factor,Power.Active.Import,Power.Active.Import.L1,Power.Active.Import.L2,Power.Active.Import.L3,Energy.Active.Import.Register,Energy.Active.Export.Register","available":"1"},"eichrecht_protocol":"none","mains_frequency":{"actual":"50000"},"voltage":{"ac":{"l1":{"actual":"23550"},"l2":{"actual":"23940"},"l3":{"actual":"23920"}}},"current":{"ac":{"l1":{"actual":"20"},"l2":{"actual":"0"},"l3":{"actual":"0"}}},"power_factor":{"ac":{"actual":"990"}},"power":{"active_total":{"actual":"50"},"active":{"ac":{"l1":{"actual":"50"},"l2":{"actual":"0"},"l3":{"actual":"0"}}}},"energy":{"active_total":{"actual":"6393240"},"active_import":{"actual":"6393240"},"active_export":{"actual":"0"}},"timestamp":"2023-12-31T07:53:47.104Z"},"emergency_shutdown":"0","rcd":{"feedback":{"available":"1"},"state":{"actual":"1"},"recloser":{"available":"0"}},"rfid":{"available":"0","authorizereq":""},"availability":{"actual":"operative"},"plug_lock":{"state":{"actual":"0","target":"0"},"error":"0"},"cp":{"pwm_state":{"actual":"1"},"state":"A","duty_cycle":"100.00"},"diode_present":"0","cable_current_limit":"-1","ev_present":"0","charging":"0","ready_for_slac":"0","ventilation":{"state":{"actual":"0"},"available":"0"},"charging_stop":"1"}}}
[lp-1  ] DEBUG 2023/12/31 08:53:53 charge total import: 6393.240kWh
[site  ] WARN 2023/12/31 08:53:53 vehicle 'evDummy' does not support automatic detection
[site  ] INFO 2023/12/31 08:53:53 site config:
[site  ] INFO 2023/12/31 08:53:53   meters:      grid ✓ pv ✓ battery ✓
[site  ] INFO 2023/12/31 08:53:53     grid:      power ✓ energy ✗ currents ✗
[site  ] INFO 2023/12/31 08:53:53     pv 1:      power ✓ energy ✗ currents ✗
[site  ] INFO 2023/12/31 08:53:53     battery 1: power ✓ energy ✗ currents ✗ soc ✓ capacity ✓
[site  ] INFO 2023/12/31 08:53:53   vehicles:
[site  ] INFO 2023/12/31 08:53:53     vehicle 1: range ✗ finish ✗ status ✗ climate ✗ wakeup ✗
[site  ] INFO 2023/12/31 08:53:53     vehicle 2: range ✓ finish ✗ status ✓ climate ✗ wakeup ✗
[site  ] INFO 2023/12/31 08:53:53     vehicle 3: range ✓ finish ✗ status ✓ climate ✓ wakeup ✓
[lp-1  ] INFO 2023/12/31 08:53:53 loadpoint 1:
[lp-1  ] INFO 2023/12/31 08:53:53   mode:        pv
[lp-1  ] INFO 2023/12/31 08:53:53   charger:     power ✓ energy ✓ currents ✓ phases ✗ wakeup ✗
[lp-1  ] INFO 2023/12/31 08:53:53   meters:      charge ✓
[lp-1  ] INFO 2023/12/31 08:53:53     charge:    power ✓ energy ✓ currents ✓
[lp-2  ] INFO 2023/12/31 08:53:53 loadpoint 2:
[lp-2  ] INFO 2023/12/31 08:53:53   mode:        pv
[lp-2  ] INFO 2023/12/31 08:53:53   charger:     power ✓ energy ✓ currents ✓ phases ✗ wakeup ✗
[lp-2  ] INFO 2023/12/31 08:53:53   meters:      charge ✓
[lp-2  ] INFO 2023/12/31 08:53:53     charge:    power ✓ energy ✓ currents ✓
[site  ] DEBUG 2023/12/31 08:53:53 set buffer soc: 90
[site  ] DEBUG 2023/12/31 08:53:53 set buffer start soc: 0
[site  ] DEBUG 2023/12/31 08:53:53 set smart cost limit: 0.14
[site  ] DEBUG 2023/12/31 08:53:53 set priority soc: 80
[lp-1  ] DEBUG 2023/12/31 08:53:53 phase timer inactive
[lp-1  ] DEBUG 2023/12/31 08:53:53 pv timer inactive
[lp-1  ] DEBUG 2023/12/31 08:53:53 guard timer inactive
[lp-2  ] INFO 2023/12/31 08:53:53 vehicle updated: unknown -> evDummy
[site  ] WARN 2023/12/31 08:53:53 interval <30s can lead to unexpected behavior, see https://docs.evcc.io/docs/reference/configuration/interval
[site  ] DEBUG 2023/12/31 08:53:53 ----
[lp-1  ] DEBUG 2023/12/31 08:53:53 charge power: 5W
[site  ] DEBUG 2023/12/31 08:53:53 pv power: 0W
[site  ] DEBUG 2023/12/31 08:53:53 battery soc: 35%
[site  ] DEBUG 2023/12/31 08:53:53 battery power: -2948W
[site  ] DEBUG 2023/12/31 08:53:53 grid power: 4144W
[site  ] DEBUG 2023/12/31 08:53:53 battery has priority at soc 35% (< 80%)
[site  ] DEBUG 2023/12/31 08:53:53 site power: 4244W
[lp-1  ] DEBUG 2023/12/31 08:53:53 charge currents: [0.02 0 0]A
[lp-1  ] DEBUG 2023/12/31 08:53:53 charge total import: 6393.240kWh
[lp-1  ] DEBUG 2023/12/31 08:53:53 charger status: A
[lp-1  ] INFO 2023/12/31 08:53:53 car disconnected
[lp-1  ] DEBUG 2023/12/31 08:53:53 set session soc limit: 0
[lp-1  ] DEBUG 2023/12/31 08:53:53 set session energy limit: 0
[site  ] DEBUG 2023/12/31 08:54:03 ----
[salia ] TRACE 2023/12/31 08:54:03 GET http://192.168.1.11/api
[salia ] TRACE 2023/12/31 08:54:03 {"device":{"product":"2310007","modelname":"Salia PLCC Slave","hardware_version":"1.0","software_version":"2.0.2","vcs_version":"V0R5c","hostname":"salia","mac_address":"00:01:87:0c:7c:41","serial":"100818241","uuid":"5491ad62-022a-4356-a32c-0001870c7c41","internal_id":"391222","ip_lo":"127.0.0.1","ip_br0:fallback":"169.254.12.53","ip_br0":"192.168.1.11"},"secc":{"port0":{"salia":{"chargemode":"manual","keyswitch":"0","mem":"335296","thermal":"48165","uptime":"1 day","thermalmax":"55509","load":"0.17","thermalmin":"0","modbus":{"status":"running"},"socketminamp":"6","socketmaxamp":"16","LM":{"internal":{"status":"internal lm not activated"}},"intctrl_limit":"16","chargedata":"-","ecoplus":"0","mains_type":"none","authmode":"free","phase_switching":{"actual":"3","status":"idle"},"tc_enabled":"0","firmwareprogress":"0","firmwarestate":"idle","status":"manual -\/-\/r -\/-\/- 0(6-16) ","limit_tc":"32","extctrl_limit":"0","pausecharging":"1","heartbeat":"alive"},"ci":{"evse":{"basic":{"grid_current_limit":{"actual":"16"},"phase_count":"3","physical_current_limit":"16","offered_current_limit":"0.0","current":{"offered":"0.0","l1":{"offered":"0.0"},"l2":{"offered":"0.0"},"l3":{"offered":"0.0"}},"power":{"offered":"0.0"}},"phase":{"actual":"0"}},"ev":{"soc":{"actual":"-1","timestamp":"2023-12-29T21:45:41.788Z"}},"charge":{"cp":{"status":"A"},"plug":{"status":"unlocked"},"contactor":{"status":"opened"},"pwm":{"status":"100.00"}}},"session":{"authorization_status":"0","authorization_method":"0"},"contactor":{"state":{"hlc_target":"0","actual":"0","target":"0"},"error":"0"},"metering":{"meter":{"serialnumber":"21030651","type":"klefr","name":"Klefr 693x\/694x","phase_count":"3","supported_measurands":"Frequency,Voltage.L1-N,Voltage.L2-N,Voltage.L3-N,Current.Import.L1,Current.Import.L2,Current.Import.L3,Power.Factor,Power.Active.Import,Power.Active.Import.L1,Power.Active.Import.L2,Power.Active.Import.L3,Energy.Active.Import.Register,Energy.Active.Export.Register","available":"1"},"eichrecht_protocol":"none","mains_frequency":{"actual":"50010"},"voltage":{"ac":{"l1":{"actual":"23540"},"l2":{"actual":"23920"},"l3":{"actual":"23920"}}},"current":{"ac":{"l1":{"actual":"20"},"l2":{"actual":"0"},"l3":{"actual":"0"}}},"power_factor":{"ac":{"actual":"1000"}},"power":{"active_total":{"actual":"50"},"active":{"ac":{"l1":{"actual":"50"},"l2":{"actual":"0"},"l3":{"actual":"0"}}}},"energy":{"active_total":{"actual":"6393240"},"active_import":{"actual":"6393240"},"active_export":{"actual":"0"}},"timestamp":"2023-12-31T07:53:57.359Z"},"emergency_shutdown":"0","rcd":{"feedback":{"available":"1"},"state":{"actual":"1"},"recloser":{"available":"0"}},"rfid":{"available":"0","authorizereq":""},"availability":{"actual":"operative"},"plug_lock":{"state":{"actual":"0","target":"0"},"error":"0"},"cp":{"pwm_state":{"actual":"1"},"state":"A","duty_cycle":"100.00"},"diode_present":"0","cable_current_limit":"-1","ev_present":"0","charging":"0","ready_for_slac":"0","ventilation":{"state":{"actual":"0"},"available":"0"},"charging_stop":"1"}}}
[lp-1  ] DEBUG 2023/12/31 08:54:03 charge power: 5W
[site  ] DEBUG 2023/12/31 08:54:03 pv power: 0W
[site  ] DEBUG 2023/12/31 08:54:03 battery soc: 35%
[site  ] DEBUG 2023/12/31 08:54:03 battery power: -3147W
[site  ] DEBUG 2023/12/31 08:54:03 grid power: 4124W
[site  ] DEBUG 2023/12/31 08:54:03 battery has priority at soc 35% (< 80%)
[site  ] DEBUG 2023/12/31 08:54:03 site power: 4224W
panic: runtime error: index out of range [1] with length 1

goroutine 170 [running]:
github.com/evcc-io/evcc/meter/tasmota.(*Connection).getPhaseValues(0xc00075d730, 0x2b56718)
	github.com/evcc-io/evcc/meter/tasmota/connection.go:246 +0x1af
github.com/evcc-io/evcc/meter/tasmota.(*Connection).Voltages(0xc0015a0a20?)
	github.com/evcc-io/evcc/meter/tasmota/connection.go:230 +0x1a
github.com/evcc-io/evcc/charger.(*Tasmota).Voltages(0xc001855400?)
	github.com/evcc-io/evcc/charger/tasmota.go:92 +0x16
github.com/evcc-io/evcc/core.(*Loadpoint).updateChargeVoltages(0xc000db8000)
	github.com/evcc-io/evcc/core/loadpoint.go:1319 +0x7d
github.com/evcc-io/evcc/core.(*Loadpoint).Update(0xc000db8000, 0x0?, 0x1, 0x0?, 0x0?, 0x0, 0xc0019851b0, 0x0)
	github.com/evcc-io/evcc/core/loadpoint.go:1516 +0x66
github.com/evcc-io/evcc/core.(*Site).update(0xc000ae6000, {0x32708b0?, 0xc000db8000})
	github.com/evcc-io/evcc/core/site.go:794 +0x5ca
github.com/evcc-io/evcc/core.(*Site).Run(0xc000ae6000, 0xc000957aa0, 0x2540be400)
	github.com/evcc-io/evcc/core/site.go:904 +0x2e5
github.com/evcc-io/evcc/cmd.runRoot.func4()
	github.com/evcc-io/evcc/cmd/root.go:281 +0x25
created by github.com/evcc-io/evcc/cmd.runRoot in goroutine 1
	github.com/evcc-io/evcc/cmd/root.go:280 +0x19b3
```


### What type of operating system are you running?

Docker container

### Version

0.123.2 (cf597421)

--- a/meter/tasmota.go
+++ b/meter/tasmota.go
@@ -56,7 +56,7 @@ func NewTasmota(uri, user, password, usage string, channels []int, cache time.Du
 	}
 
 	var currents, voltages func() (float64, float64, float64, error)
-	if usage != "grid" {
+	if usage != "grid" && len(channels) == 3 {
 		currents = c.currents
 		voltages = c.voltages
 	}

